site/v3-docs/solid/intro.md

```diff
@@ -0,0 +1,24 @@
+<p>
+  <img src="https://assets.solidjs.com/banner?project=Library&type=solid-lightning" alt="SolidJS Lightning" />
+</p>
+
+# SolidJS for Lightning 3
+
+Solid-Lightning is a UI framework for the [Lightning 3 Renderer](https://github.com/lightning-js/renderer) built with [SolidJS](https://www.solidjs.com/) [Universal Renderer](https://github.com/solidjs/solid/releases/tag/v1.2.0). It allows you to declaratively construct lightning nodes with reactive primitives, just as you would construct a DOM tree in SolidJS.
```

Comment:
I dont think that last part is true, you're not constructing "just as you would construct a DOM tree". The dom primitives won't work when creating a SolidJS lightning app.

Reply:
👍